cmd/root.go

```diff
@@ -124,6 +114,7 @@ func setupLogger(atmosConfig *schema.AtmosConfiguration) {
 	}
 
 	log.SetOutput(output)
+	log.Debug("Log level set to", "level", log.GetLevel(), "file", atmosConfig.Logs.File)
```

Comment:
Weird the “level=DEBU” is not in the output.

Reply:
Yep that was what i thought as well. Do we want to dig into it?

Reply:
Yes, might be worth an investigation. Anytime something doesn't work as expected, we should wonder why.